libraries/cyclestreets-view/src/main/java/net/cyclestreets/views/overlay/WaymarkOverlay.kt

```diff
@@ -120,10 +134,13 @@ class WaymarkOverlay(private val mapView: CycleMapView) : Overlay(), PauseResume
             postTranslate(screenPos.x.toFloat(), screenPos.y.toFloat())
         }
 
+        val x = screenPos.x.toFloat()
+        val y = screenPos.y.toFloat()
         canvas.apply {
             save()
-            rotate(-projection.orientation, screenPos.x.toFloat(), screenPos.y.toFloat())
+            rotate(-projection.orientation, x, y)
             drawBitmap(bitmap, bitmapTransform, bitmapPaint)
+            drawText(waymarkPosition, x - halfWidth/10, (y - halfHeight/1.8).toFloat(), boldTextBrush)
```

Comment:
Magic number 1.8 here.  Should this be `INCREASE_WAYMARK_SIZE` (i.e. currently 1.5) or a new constant?
Also what's with `halfWidth/10`?

Reply:
IIRC both of those numbers are to position the text within the waymark.  Let me try and think of appropriate constant names... 
HORIZONTAL_TEXT_POSITION_ADJUSTMENT, VERTICAL_TEXT_POSITION_ADJUSTMENT or something like that.

Reply:
Those constants are a bit of a mouth / eye-ful, so I've created more descriptive variables instead.

Reply:
Also I would extract all constants to top level for the class - not inline in specific functions/methods as you have currently.

Am ready to re-review your update when it is good to go - ping me at that time please.

Reply:
OK, done, and new issue raised for the bitmaps - #498